src/app/components/MediaLoader/types.ts

```diff
@@ -55,6 +57,8 @@ export type ConfigBuilderProps = {
   translations?: Translations;
   adsEnabled?: boolean;
   showAdsBasedOnLocation?: boolean;
+  embedUrl?: string;
+  isAmp: boolean;
```

Comment:
We may be getting rid of this, but if we are not, could it be optional and defaulted to false?

Reply:
Have removed!